src/proptest_strategies.rs

```diff
@@ -94,6 +95,7 @@ prop_compose! {
             PayloadContent::ControlMsg(control_type, _) => MessageType::Control(control_type.clone()),
             PayloadContent::Verbose(_) => MessageType::Log(LogLevel::Warn),
             PayloadContent::NonVerbose(_, _) => MessageType::Log(LogLevel::Debug),
+            PayloadContent::NetworkTrace(_) => todo!()
```

Comment:
This feels like a forgotten `todo`. 
If this is intended, then I think a small comment here would be nice

Reply:
Yes, that was a forgotten todo!